metaflow/metaflow_config.py

```diff
@@ -68,6 +102,31 @@ def from_conf(name, default=None):
     if from_conf("METAFLOW_DATASTORE_SYSROOT_S3")
     else None,
 )
+
+DATATOOLS_DEFAULT_CLIENT_PARAMS = from_conf(
+    "METAFLOW_DATATOOLS_CLIENT_PARAMS",
+    '{"endpoint_url": %s, "verify": %s}' % (S3_ENDPOINT_URL, S3_VERIFY_CERTIFICATE)
```

Comment:
I think you'd want to to quote `endpoint_url`. Also there gotta be a more robust way to construct that default value without raw string formatting, idk `json.dumps()` maybe

Reply:
or rather: 
```
json.dumps({"endpoint_url": S3_ENDPOINT_URL, "verify": S3_VERIFY_CERTIFICATE})
```